index.js

```diff
@@ -366,7 +367,7 @@ input UpdateAnswerInput {
   secondaryLabel: String 
   qCode: String
   type: AnswerType
-  mandatory: Boolean
+  properties: JSON
```

Comment:
Will this update all of the answer properties whenever any single property is updated?

Did you consider adding a new mutation to allow properties to be updated independently i.e. 

```
updateProperty(answerId, key, value)
```
and if so what was the reason behind taking this approach?

Reply:
It updates all properties at once replacing existing JSON with new values. 

Decided not to separate to its own mutation to reduce the code/extra complexity required in the API to handle merging values and to follow existing pattern of mutations. Although, I see the value of smaller updates.  Do you recommend refactoring to handle independent property updates?

Reply:
I don't think it's necessary to refactor right now as there's only a handful of properties. It's something we can change later if we find there is any degradation to performance.